.github/workflows/main.yml

```diff
@@ -0,0 +1,124 @@
+       run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
+     
+  robolectric_tests:
+    name: Robolectric Tests (Non-App Modules)
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04] 
+    steps:
+     - uses: actions/checkout@v2
+     - uses: actions/cache@v2
+       id: cache
+       with:
+         path: ~/.gradle
+         key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}-jars-{{ checksum "build.gradle" }}
+         
+     - name: Set up JDK 1.9
+       uses: actions/setup-java@v1
+       with:
+          java-version: 1.9
+
+     - name: Install Dependencies
+       if: steps.cache.outputs.cache-hit != 'true'
+       run: ./.circleci/gradle/gradlew androidDependencies
+     
+     - name: Build App
+       # We require 'sudo' to avoid an error of the existing android sdk. See https://github.com/actions/starter-workflows/issues/58
+       run: sudo ./.circleci/gradle/gradlew assembleDebug
+    
+     - name: Utility tests
+       # We require 'sudo' to avoid an error of the existing android sdk. See https://github.com/actions/starter-workflows/issues/58
+       run:  sudo ./.circleci/gradle/gradlew :utility:testDebugUnitTest
+     - name: Upload Utility Test Reports
+       uses: actions/upload-artifact@v2
+       if: ${{ always() }} # IMPORTANT: Upload reports regardless of status
+       with:
+         name: utility reports
+         path: utility/build/test-results
+     
+     - name: Domain tests
+       # We require 'sudo' to avoid an error of the existing android sdk. See https://github.com/actions/starter-workflows/issues/58
+       run:  sudo ./.circleci/gradle/gradlew :domain:testDebugUnitTest
+     - name: Upload Domain Test Reports
+       uses: actions/upload-artifact@v2
+       if: ${{ always() }} # IMPORTANT: Upload reports regardless of status
+       with:
+         name: domain reports
+         path: domain/build/test-results
+  
+  app_tests:
+    name: Robolectric Tests - App Module (Non-Flaky)
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps: 
+      - uses: actions/checkout@v2
+      
+      - uses: actions/cache@v2
+        with:
+         path: ~/.gradle/caches
+         key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}-jars-{{ checksum "build.gradle" }}
+         
+      - name: Set up JDK 1.9
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.9
+      
+      - name: Robolectric tests - FAQ, Help, Mydownloads, Parser, ProfileProgress, RecyclerView, Story, Utility tests
+        # We require 'sudo' to avoid an error of the existing android sdk. See https://github.com/actions/starter-workflows/issues/58
+        run: |
+          sudo ./.circleci/gradle/gradlew :app:testDebugUnitTest --tests org.oppia.app.faq* --tests org.oppia.app.help* --tests org.oppia.app.mydownloads* \
```

Comment:
There is backward slash at the end of this line. Does this indicate something?

Reply:
It is to indicate that the command spans more than one line

Reply:
Could you just put it all on one line? We typically try never to end lines with backslashes (because it's hard to e.g. tell if there's a space after the backslash).

Reply:
Okay sure, done

Reply:
Yes, this make sense. Thanks!